mkt/developers/templates/developers/terms_standalone.html

```diff
@@ -0,0 +1,29 @@
+<html>
+  <head>
+    {{ css('mkt/devreg') }}
+    <style type="text/css">
```

Comment:
This is sort of lame, but some of the CSS is very specific to devhub. This is just enough for a nice, iframe-able presentation:

<img width="1031" alt="screen shot 2015-09-21 at 12 16 24 pm" src="https://cloud.githubusercontent.com/assets/23885/10000637/a0791dd6-605a-11e5-84f6-979888ee49b7.png">